gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -204,8 +212,20 @@ public void execute() throws MojoExecutionException {
 
         } else {
           Collection<String> simulations = simulations();
+          boolean buildFails = false;
           for (String simulation : simulations) {
-            executeGatling(jvmArgs, gatlingArgs(simulation), testClasspath, toolchain);
+            try {
+              executeGatling(jvmArgs, gatlingArgs(simulation), testClasspath, toolchain);
+            } catch (Exception e) {
+              if (!continueAfterFail) {
```

Comment:
Please fast exit first when there's no pending simulation after this loop iteration.